ast2vec/repo2/source.py

```diff
@@ -24,9 +20,13 @@ def convert_uasts(self, file_uast_generator):
 
         for file_uast in file_uast_generator:
             sources = self._get_sources(file_uast.filepath)
-            src_codes.append(sources)
-            uasts.append(file_uast.response)
-            filenames.append(file_uast.filename)
+            if sources is not None:
+                src_codes.append(sources)
+                uasts.append(file_uast.response)
+                filenames.append(file_uast.filename)
+
+        assert len(src_codes) == len(uasts)
```

Comment:
Assertion is smth which is a bug in your code. Here you validate the file which comes from the outer world, so you should throw an exception instead. `ValueError` I guess